samples/core/use_run_info/use_run_id.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import kfp
+import kfp.dsl as dsl
+
+
+def get_run_info(run_id: str):
+    '''Example of getting run info for current pipeline run'''
+    import kfp
+    print(f'Current run ID is {run_id}.')
+    # KFP API server is usually available as ml-pipeline service in the same
+    # namespace, but for full Kubeflow deployment, you need to edit this to
+    # http://ml-pipeline.kubeflow:8888, because your pipelines are running in
+    # user namespaces, but the API is at kubeflow namespace.
+    client = kfp.Client(host='http://ml-pipeline:8888')
+    run_info = client.get_run(run_id=run_id)
+    # Hide verbose info
+    print(run_info.run)
+
+
+get_run_info_component = kfp.components.create_component_from_func(
+    func=get_run_info,
+    packages_to_install=['kfp'],
```

Comment:
(No action needed for this PR) This just reminds me that we might want to move `dsl.RUN_ID_PLACEHOLDER` out to a standalone pkg as well.

Reply:
note, I'm adding kfp package here, because I use kfp SDK client inside the component. It'll probably also be better if kfp.Client is a standalone pkg.